### DISCUSSION TITLE
Does Thorium support hyphenation?

### DISCUSSION THREAD

**Discussion by beedaddy** [edrlab/thorium-reader#1842] [Q&A]
I can't find a corresponding switch in the settings. With EPUBs, with `hyphens: auto;` etc. in the CSS I can't observe hyphenation either. Since I consider this an essential feature, I'd be interested to know if this is on the to-do list, or if it was left off for reasons.

**Comment by danielweck** [comment]
It would be interesting to know how this would interact with ReadiumCSS: https://github.com/search?q=repo%3Areadium%2Freadium-css%20hyphens&type=code
